2.cellprofiler_analysis/scripts/nf1_analysis.py

```diff
@@ -20,16 +20,20 @@
 # In[2]:
 
 
-path_to_pipeline = pathlib.Path("NF1_analysis.cppipe").absolute()
-
-path_to_output = pathlib.Path("analysis_output").absolute()
+path_to_output = pathlib.Path("./analysis_output").resolve(strict=True)
 
 plates_info_dictionary = {
     "Plate_1": {
-        "path_to_images": pathlib.Path("../1.cellprofiler_ic/Corrected_Plate_1/").absolute(),
+        "path_to_pipeline" : pathlib.Path("NF1_analysis_plate1_plate2.cppipe").resolve(strict=True),
```

Comment:
Should this reference `plate1` in the name somehow? If not, consider making a comment about the numbering so as to provide context on why `plate2` is referenced from the `Plate_1` key.

```suggestion
        "path_to_pipeline" : pathlib.Path("NF1_analysis_plate1_plate1.cppipe").resolve(strict=True),
```

Reply:
I will add a comment, essentially I needed two different cppipe files for plates 1 and 2 and then for plates 3 and 3' so I had the name reflect that. I agree to make it clearer so thank you for bringing it up!